core/primitives/src/views.rs

```diff
@@ -197,7 +198,7 @@ impl From<AccessKeyView> for AccessKey {
 }
 
 /// Set of serialized TrieNodes that are encoded in base64. Represent proof of inclusion of some TrieNode in the MerkleTrie.
-pub type TrieProofPath = Vec<String>;
+pub type TrieProofPath = Option<Vec<TrieProofItem>>;
```

Comment:
Adding Option will break serialisation compatibility. This needs to remain a Vec.

Reply:
Not sure what to do. I thought this was the right way to solve the issue of returning an `Option` and exposing it into the RPC. What do you suggest @mina86

Reply:
yep - Vec<String> has to stay (otherwise a bunch of View calls will no longer be deserializable).